### PR TITLE
Godot: Bump to Godot 2.1.6

### PR DESCRIPTION
**Pull request by kenmays**
godot-platform

### CODE REVIEW

File: dev-games/godot/godot-2.1.6.recipe [hunk 1]
```diff
@@ -7,11 +7,11 @@ click to a number of platforms, including the major desktop platforms \
 (Linux, Mac OSX, Windows) as well as mobile (Android, iOS) and web-based \
 (HTML5) platforms."
 HOMEPAGE="https://godotengine.org/"
-COPYRIGHT="2007-2017 Juan Linietsky, Ariel Manzur"
+COPYRIGHT="2007-2019 Juan Linietsky, Ariel Manzur"
```
Comment (extrowerk) [inline dev-games/godot/godot-2.1.6.recipe]: Check your screenshot

**Comment by extrowerk** [comment]
I don't think it closes #3453

File: dev-games/godot/godot-2.1.6.recipe (hunk 1 above)
Comment (kenmays) [reply dev-games/godot/godot-2.1.6.recipe]: Yes, copyright is correct and I'm using godot 2.1.6,. Screenshots were done with 2.1.5 on same machine prior until I replaced it with the 2.1.6 build.

**Comment by extrowerk** [comment]
We tell you all the time: "Even after so many time, we still can't be 
sure about your PRs."
We even asked proof screenshots, you provided them, cool, at least we 
see, you got it compiled.
But now you just told your proof is compromitted, because it is not 
even about the version what mentioned in your PR.
What do you think about this and about the situation? How should we 
handle this? Please, tell us, because some of us are really puzzled. 
There must be a way which makes everybody happy, but it is clear we 
haven't found it yet. I am seriously interested in your 
explanation/motivation, so tell us: what are the problems which doesn't 
lets you shine. Maybe we do something wrong.

This is 100% private opinion, btw.

kenmays <notifications@github.com>:
> kenmays commented on this pull request.
> 
> 
> 
> > @@ -7,11 +7,11 @@ click to a number of platforms, including the 
> > major desktop platforms \
>  (Linux, Mac OSX, Windows) as well as mobile (Android, iOS) and web-
> based \
>  (HTML5) platforms."
>  HOMEPAGE="https://godotengine.org/"
> -COPYRIGHT="2007-2017 Juan Linietsky, Ariel Manzur"
> +COPYRIGHT="2007-2019 Juan Linietsky, Ariel Manzur"
> 
> Yes, copyright is correct and I'm using godot 2.1.6,. Screenshots 
> were done with 2.1.5 on same machine prior until I replaced it with the 
2.1.6 build.

**Comment by kenmays** [comment]
Read what I said in chat window. godot 3.x doesn't work. Please, let us just fix the issue at hand.

**Comment by extrowerk** [comment]
You change the screenshots and your comments all the time, i know about the problems with godot3, but would you answer my questions?

**Comment by kenmays** [comment]
Answered you in IRC....

File: dev-games/godot/godot-2.1.6.recipe (hunk 1 above)
Comment (kenmays) [reply dev-games/godot/godot-2.1.6.recipe]: Fixed copyright

**Comment by kenmays** [comment]
extrowerk - Simple. The key is that we help one another. You asked about my previous screenshot versus the current PR not matching. I stated in IRC that it was a minor bug fix, not a major one so there was nothing major changed. The program works as you did previous testing of the same major version (these are just minor bump updates).  Different than posting Godot3 which does not work properly (you state you knew this beforehand, but posted the PR yourself). So, why question my integrity when their is a lot of Haiku code or PRs that are 'suspect or poorly tested' (same for other  professional or free OSes/kernels) - but I actually posted the screenshot, tested it, and proofed the compile on BOTH x86 and x64. Yet, we want to squabble over rebase or squashing with git (hahaha).

Understand about the past issues with my PRs, but I've gotten better with Haiku and Haikuports. So, point taken over here. No harm done or lives lost. Can we just move along and be friendly?

**Comment by kenmays** [comment]
Removed. Deemed not needed by waddlesplash.

**Comment by waddlesplash** [comment]
That is most emphatically not what I said.

**Pull request merged**